app/controllers/users/emails_controller.rb

```diff
@@ -12,14 +12,18 @@ class EmailsController < ApplicationController
 
     def show
       analytics.add_email_visit
+      user_session[:from_select_email_flow] = params[:from_select_email_flow]
```

Comment:
Why do we store this in the session?

And part of the reason for asking is that I think there's potential for confusion to have a `:from_select_email_flow` in both `session` and `user_session` that each serves different purposes

Reply:
It was to ensure we could add it as an attribute for the post action, but might be a good idea to to have it be a hidden field value that submits as a parameter.

Reply:
Yeah I think I see what you're getting at with that, but it might not work as expected other than the first time, since the second time it'd be set to `nil` if the parameter isn't present, and we're not falling back to the existing session value.

I think a hidden form parameter could work.